packages/ui/src/SvgIcon/SvgIcon.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { expect } from '@jest/globals'
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import SVgIcon from './index'
+import { Default as story } from './index.stories'
+
+describe('SVgIcon', () => {
```

Comment:
Small Typo the import and test name should be `SvgIcon`. Watch out for V is currently upper caps.